cloudformation_seed/cfn_stackset.py

```diff
@@ -496,6 +497,22 @@ def rollout_stackset(self):
         else:
             self.rollout_accounts()
 
+    def check_stack_instances(self):
+        failures = []
+        c = util.session.client('cloudformation')
+        for page in c.get_paginator('list_stack_instances').paginate():
+            for summary in page['Summaries']:
```

Comment:
Is this all instances of all StackSets or the StackSet being deployed?

Reply:
yea, this is my miss. will update this to filter into stackset parameter